content/building/yaml.md

```diff
@@ -360,3 +360,7 @@ For example, you can write a build script with Dart like this:
         app_id: '...'                     # App's unique identifier in App Store Connect
         apple_id: name@example.com        # Email address used for login
         password: Encrypted(...)          # App-specific password
+
+### Example
+
+If you want to know more about using YAML, there is a complete example of building a native iOS app with a Flutter module <a href="https://blog.codemagic.io/how-to-add-flutter-modules-to-native-ios-project-and-test-it-on-codemagic/" target="_blank" onclick="sendGtag('Link_in_docs_clicked','dynamic-workflows-with-codemagic-api')">here</a>. This article also describes how to use YAML to test this app on Codemagic CI/CD.
```

Comment:
there is a copy-past in all google analytics events. It sends 'dynamic-workflows-with-codemagic-api' instead of proper names